programs/psinode/main.cpp

```diff
@@ -1427,6 +1541,11 @@ void to_config(const PsinodeConfig& config, ConfigFile& file)
           "", "admin-authz", admin_authz, [](std::string_view text) { return std::string(text); },
           "Authorization for admin access");
    }
+   if (config.http_timeout != Timeout::none())
+   {
+      file.set("", "http-timeout", to_string(config.http_timeout),
+               "The maximum time for HTTP clients to send or receive a message");
```

Comment:
I think something like "The timeout before the server closes idle HTTP connections" is a clearer description.